application/src/test/java/org/togetherjava/tjbot/jda/JdaTester.java

```diff
@@ -20,9 +19,7 @@
 import net.dv8tion.jda.api.interactions.callbacks.IReplyCallback;
 import net.dv8tion.jda.api.interactions.components.ItemComponent;
 import net.dv8tion.jda.api.interactions.components.LayoutComponent;
-import net.dv8tion.jda.api.requests.ErrorResponse;
-import net.dv8tion.jda.api.requests.Response;
-import net.dv8tion.jda.api.requests.RestAction;
+import net.dv8tion.jda.api.requests.*;
```

Comment:
No star imports please

Reply:
actually ig spotless did that, there are star imports in the codebase already

Reply:
It is probably not spotless but intellij